### PR TITLE
Add sentry integration

### PR DESCRIPTION
**Pull request by sydorovdmytro**
Sentry integration to track all traces in one place. Requires https://github.com/raccoongang/configuration/pull/31/

### CODE REVIEW

File: settings.py [hunk 1]
```diff
@@ -51,3 +51,11 @@
     GRADER_ROOT = ENV_TOKENS.get('GRADER_ROOT')
 
     SANDBOX_PYTHON = ENV_TOKENS.get('SANDBOX_PYTHON', '/opt/edx/bin/sandbox-python')
+
+
+if ENV_TOKENS.get(SENTRY_DNS):
+    import raven
```
Comment (OPersian) [inline settings.py]: Could you please move the import statement to the beginning of the file? Thanks

**Review by OPersian — APPROVED** [review]
Please refer to a single comment

File: settings.py [hunk 2]
```diff
@@ -51,3 +51,11 @@
...
+    INSTALLED_APPS = INSTALLED_APPS + ( 'raven.contrib.django.raven_compat', )
```
Comment (cmltaWt0) [inline settings.py]: Please use this syntax:
```
INSTALLED_APPS += (
    'raven.contrib.django.raven_compat',
 )
```

Comment (sydorovdmytro) [reply settings.py]: Done

File: settings.py (hunk 1 above)
Comment (sydorovdmytro) [reply settings.py]: Done

**Pull request merged**